src/llmcompressor/transformers/compression/sparsity_config.py

```diff
@@ -20,7 +26,7 @@ class SparsityConfigMetadata:
     metadata from the model
     """
 
-    SPARSITY_THRESHOLD: float = 0.4
+    SPARSITY_THRESHOLD: float = 0.5
```

Comment:
Does this have the potential to block 2:4 sparsity since it is exactly 0.5 (in theory)

Reply:
We check for `>=` but made this `0.49` since it makes sense.